Sources/Rendering/Core/Camera/index.d.ts

```diff
@@ -0,0 +1,302 @@
+import VtkObject from '../../../macro';
```

Comment:
can you use the absolute path? Or is it not possible as the alias are not understood by ts?

Reply:
Yes i can use the absolute path